Mathlib/AlgebraicGeometry/Modules/Sheaf.lean

```diff
@@ -4,24 +4,28 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Joël Riou
 -/
 
-import Mathlib.Algebra.Category.ModuleCat.Sheaf
+import Mathlib.Algebra.Category.ModuleCat.Sheaf.Abelian
 import Mathlib.AlgebraicGeometry.Modules.Presheaf
 
 /-!
 # The category of sheaves of modules over a scheme
 
-In this file, we define the category of sheaves of modules
+In this file, we define the abelian category of sheaves of modules
 `X.Modules` over a scheme `X`.
 
 -/
 
 universe u
 
+open CategoryTheory
+
 namespace AlgebraicGeometry.Scheme
 
 variable (X : Scheme.{u})
 
 /-- The category of sheaves of modules over a scheme. -/
 abbrev Modules := SheafOfModules.{u} X.ringCatSheaf
 
+noncomputable instance : Abelian X.Modules := inferInstance
```

Comment:
🎉

Reply:
Actually, I have removed it because I am not completely sure what happens when two (defeq) `instance` declarations are made. But, at least, if `AlgebraicGeometry.Modules.Sheaf` and `ModuleCat.Sheaf.Abelian` are imported, this will be found!

Reply:
I believe it doesn't have bad effects as `inferInstance` is an `abbrev`. In fact it even helps because typeclass inference no longer needs to jump through hoops.

Reply:
Ok, I have added it back...